pom.xml

```diff
@@ -146,7 +89,8 @@
                 <scope>provided</scope>
             </dependency>
 
-            <!-- OSGi Dependencies -->
+            <!-- OSGi DEPREACTED Dependencies -->
```

Comment:
Not a big deal, but `deprecated` is misspelled here.

Reply:
Done